kolibri/core/assets/src/views/AppBarCorePage.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+
+  <!-- TODO useScrollPosition to set scrollPosition...
+    here or in router, but somewhere -->
+  <div>
+    <ScrollingHeader :scrollPosition="0">
+      <AppBar
+        ref="appBar"
+        :title="title"
+        :hideNavBar="hideNavBar"
```

Comment:
getting a console error undefined for this

Reply:
Fix pushed for this